go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/stretchr/testify v1.8.1
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.20.0
+	go.mongodb.org/atlas v0.21.0
```

Comment:
we have recently upgraded atlas go client in atlascli. Can you update your PR with the latest changes from the repo?

Reply:
Done